components/server/BUILD.yaml

```diff
@@ -83,10 +83,44 @@ packages:
         build: ["npx", "tsc"]
 scripts:
   - name: telepresence
+    deps:
+      - dev/telepresence-hack:copy-mounts
     script: |
+      set -e
       export GITPOD_LAYER_NAME=`git log -n 1 --pretty=format:%H -- ../../devops/images/workspace-image-builder/`
 
-      telepresence --swap-deployment server \
-                   --method vpn-tcp \
-                   --run yarn start-ee-inspect | \
-        leeway run components:dejson-log-output
+      export DST=$(mktemp -d)
+      echo Preparing filesystem in $DST
+      mkdir -p $DST/mounts
+      echo $(which copy-mounts)
+      $DEV_TELEPRESENCE_HACK__COPY_MOUNTS/telepresence-hack --destination=$DST/mounts --deployment=server
+
+      cat > $DST/patch.yaml <<EOF
+      spec:
+        template:
+          spec:
+            containers:
+            - name: server
+              securityContext:
+                privileged: true
+                runAsUser: 0
+            serviceAccountName: ws-daemon
+      EOF
+      kubectl patch deployments.apps server --patch-file $DST/patch.yaml
```

Comment:
last time I tried to `kubectl patch` the server deployment, it did override the actual deployment, leaving the content of the patch only.

I ended up with patching like this: https://kubernetes.io/docs/reference/kubectl/cheatsheet/#patching-resources

Reply:
Patching the way I did in this PR works because the `container` in a [PodSpec](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#podspec-v1-core) have `merge` as `kubernetes-patch-strategy` (see [here for details](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/#notes-on-the-strategic-merge-patch))